model.py

```diff
@@ -66,7 +71,7 @@ def train_doc2vec(corpus):
     # 10 epochs take around 10 minutes on my machine (i7), if you have more time/computational power make it 20
     for epoch in range(10):
         logging.info('Training iteration #{0}'.format(epoch))
-        d2v.train(corpus, total_examples=d2v.corpus_count, epochs=d2v.iter)
+        d2v.train(corpus, total_examples=d2v.corpus_count, epochs=d2v.epochs)
```

Comment:
- Shall we reverse the changes that won't break the script (except of this line, because of gensim update). and leave the other modifications to a separate PR after merging this one? 
- To clarify, let's leave this PR for Travis and pyup integrations, and needed gesnim syntax modifications.

Reply:
Cool then I will revert all the changes except for requirements.txt

Reply:
except of this line, because using `iter` instead of `epochs` with the new gensim version will fail, right?

Reply:
Yes, it will throw a deprecation warning.